sagan/train.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+import tensorflow as tf
+from tensorflow import keras
+
+from models.generator import build_generator
+from models.discriminator import build_discriminator
+
+
+## Define the hyperparameters
```

Comment:
Hyperparameters should be defined separately in a config file. Refer to https://github.com/soumik12345/point-cloud-segmentation/blob/main/configs/shapenetcore.py